.changeset/great-cherries-fly.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-xcmetrics': minor
```

Comment:
Isn't this still at `patch` release as it doesn't look like it breaks the API and the current version is `<1.0.0` so `patch === minor`?